AnujNaik/lab1/part2/src/part2.cpp

```diff
@@ -0,0 +1,9 @@
+// #include <part2.h>
+#include "../include/part2.h"
```

Comment:
your commented code is correct, the relative include is incorrect. this is set up for you by your cmake